association-backend/src/association_membership/domain/entities.py

```diff
@@ -62,40 +68,81 @@ def _change_state(self, to: SubscriptionStatus):
         logger.info("Switching subscription from status %s to %s", self.status, to)
         self.status = to
 
+    def add_stripe_subscription_payment(
```

Comment:
What about `add_pretix_payment`, is for another PR?

Reply:
Yes, I added the model already (which I might regret) but don't want to add more since we don't really know how the pretix integration looks like